basis/io/sockets/secure/openssl/openssl.factor

```diff
@@ -113,9 +125,13 @@ M: rsa dispose* handle>> RSA_free ;
 
 : generate-eph-rsa-key ( ctx -- )
     [ handle>> ]
-    [
-        config>> ephemeral-key-bits>> RSA_F4 f f RSA_generate_key
-        dup ssl-error <rsa> &dispose handle>>
+    [| ctx |
+        RSA_new :> rsa-struct
```

Comment:
I'm not happy with this word, but I wasn't terribly clear how to shorten it meaningfully/get rid of the use of `[|`, either, so I picked some unorthodox formatting so I could at least be clear what was operating on what. Totally happy to alter this as you want.